templates/init/dataconnect/connector.yaml

```diff
@@ -5,9 +5,11 @@ authMode: "PUBLIC"
 # generate:
 #   javascriptSdk:
 #     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
-#     package: "@firebasegen/my-connector"
+#     package: "@firebasegen/default"
 #     packageJSONDir: < Optional. Path to your Javascript app's package.json>
 #   swiftSdk:
 #     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
+#     package: "firebasegen/default"
 #   kotlinSdk:
 #     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
+#     package: com.google.firebase.dataconnect.connectors.default
```

Comment:
Denver suggests to change it to

`connectors.default`

Reply:
https://github.com/firebase/firebase-tools/blob/master/src/init/features/dataconnect/sdk.ts#L127

BTW, it's already changed here

Reply:
+1 to fredzqm's comment above

Reply:
Do we still need this line in this pr then?
`com.google.firebase.dataconnect.connectors.default` is what was generated when I ran `firebase init dataconnect:sdk`